bebop_driver/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(bebop_driver)
 
+add_definitions(-std=c++11 -D__STDC_CONSTANT_MACROS)
```

Comment:
We should find an alternative to using `c++11` since [it's not allowed for packages targeted for ROS _Indigo_](http://www.ros.org/reps/rep-0003.html#indigo-igloo-may-2014)